migration/db/schema.rb

```diff
@@ -10,7 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
```

Comment:
conflict message

Reply:
oh...thanks!
The new migration has been done just now.